pkg/apis/controller/apis.go

```diff
@@ -14,12 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Generate deepcopy, clientset, listers, informers for apis
-//go:generate ../../../hack/update-codegen.sh
-
-// Generate open-api for apis
-//go:generate ../../../hack/update-openapigen.sh
-
```

Comment:
@tenzen-y Why did we delete it ?

Reply:
I moved this script to https://github.com/kubeflow/katib/blob/42fe2783ed7124f7a3c6717eb902af0a1ef7abec/pkg/apis/apis.go because we need to generate codes for the `pkg/apis/controller` and `pkg/apis/config`.

Reply:
Sounds good, thanks!